### PR TITLE
Support better datepicker

### PR DESCRIPTION
**Pull request by olafura**

### CODE REVIEW

File: src/plugins/react/reactquestiondate.tsx [hunk 1]
```diff
@@ -13,28 +14,20 @@ export default class SurveyQuestionDate extends React.Component<any, any> {
         this.state = { value: this.question.value };
         this.handleOnChange = this.handleOnChange.bind(this);
     }
-    handleOnChange(event) {
-        this.question.value = event.target.value;
+    handleOnChange(date) {
+        this.question.value = date.format("MM/DD/YYYY");
```
Comment (robertpeacock22) [inline src/plugins/react/reactquestiondate.tsx]: Do we not want this as DD/MM/YYYY?

**Review by robertpeacock22 — CHANGES_REQUESTED** [review]

File: src/plugins/react/reactquestiondate.tsx (hunk 1 above)
Comment (olafura) [reply src/plugins/react/reactquestiondate.tsx]: I'm all for that

**Review by robertpeacock22 — APPROVED** [review]

File: src/plugins/react/reactquestiondate.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ﻿import * as React from 'react';
 import QuestionDateModel from "../question_date";
 import {ReactQuestionFactory} from "../../react/reactquestionfactory";
-
+import DatePicker from 'react-datepicker';
+import moment from 'moment';
```
Comment (mariusbutuc) [inline src/plugins/react/reactquestiondate.tsx:5]: Where does `moment` come from?

File: package.json [hunk 3]
```diff
@@ -51,13 +51,15 @@
...
     "live-server": "1.2.0",
+    "moment": "^2.18.1",
     "node-sass": "4.4.0",
     "node-uuid": "1.4.7",
     "phantomjs-prebuilt": "2.1.14",
     "preact": "7.2.0",
     "preact-compat": "3.13.1",
     "qunitjs": "2.2.0",
     "react": "15.0.1",
+    "react-datepicker": "^0.48.0"
```
Comment (mariusbutuc) [inline package.json]: Does there need to be a `,` here?

Comment (olafura) [reply package.json]: Done ;)

**Review by mariusbutuc — APPROVED** [review]
🏍

**Pull request merged**